todo-backend/app/Services/TodoService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+
+namespace App\Services;
+
+use App\Todo as Todo;
+use Illuminate\Support\Facades\Auth;
+use App\Http\Requests\StoreTodos;
+use App\Http\Requests\UpdateTodos;
+use \Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+
+class TodoService
+{
+    use AuthorizesRequests;
+    public function getAll()
+    {
+        return Auth::user()->todos;
+    }
+
+    public function getOne(Todo $todo)
+    {
+        $this->authorize('view', $todo);
```

Comment:
e da, zaboravio sam da ti pomenem
servis ne bi trebalo da zna za http layer (u prevodu bilo sta sto je vezano za request, auth...)

tako da bi autorizacija trebalo da ostane u kontroleru

Reply:
razlog za to je sto logika iz servisa treba da moze da se koristi u bilo kom momentu, a ne samo dok je request aktivan
kasnije ces videti kako queue radi, on nema veze sa requestom a moze da poziva servis